technical-reports/format/groups.md

```diff
@@ -206,6 +206,36 @@ In this example, the "brand" group has 2 extensions: `org.example.tool-a` and `o
 
 </aside>
 
+### Deprecated
+
+The optional **`$deprecated`** property is a boolean or string where tooks MAY specify all tokens within the group are deprecated. A token MAY be marked deprecated in any of the following scenarios:
```

Comment:
Same typo here.

```suggestion
The optional **`$deprecated`** property is a boolean or string where tools MAY specify all tokens within the group are deprecated. A token MAY be marked deprecated in any of the following scenarios:
```

(aside: Maybe we should consider having a single chapter for all token and/or group properties, since many exist for both. Each property could specify if it can be used on tokens, groups or both. That could avoid a lot of the duplication and copy-pasting we need to do today. Obvs. that would need to be a separate PR)

Reply:
>(aside: Maybe we should consider having a single chapter for all token and/or group properties, since many exist for both.

That would make sense to me. It could also possibly clarify [some in-the-weeds confusion around parsing token vs group nodes](https://github.com/design-tokens/community-group/issues/253)